imports/ui/components/ChainStates.jsx

```diff
@@ -53,7 +56,10 @@ export default class ChainStates extends Component{
         if (this.props.coinStats != prevProps.coinStats){
             if (this.props.coinStats.usd){
                 this.setState({
-                    price: numbro(this.props.coinStats.usd).format("$0,0.00"),
+                    price: numbro(this.props.coinStats.usd).formatCurrency({
```

Comment:
Thanks! This was fixed in the latest patch to read the value directly from `coingecko`. This is a good customization though. Will merge if you rebase to `v0.37.x-patch-4`.